draft-welzl-ccwg-ratelimited-increase.md

```diff
@@ -128,8 +128,10 @@ Thus, such an increase is allowed by the second rule.
 
 # Security Considerations
 
-While congestion control issues could result in unwanted competing traffic, they do not directly result in security considerations. Transport protocols that provide authentication (including those using encryption) or that are carried over protocols that provide authentication, can protect the congestion control mechanisms from network attack.
+Transport protocols that provide authentication (including those using encryption) or that are carried over protocols that provide authentication,
```

Comment:
```suggestion
Transport protocols that provide authentication (including those using encryption) or that are carried over protocols which provide authentication,
```

From what I once learned, "which" is right when the clause after it supplies extra information (rather than limiting a set). So... wouldn't it be ok here?  and this would avoid the triple use of "that".

Reply:
I see, let's try and get this untangled, I'll rewrite.